src/app/molecules/message/Message.jsx

```diff
@@ -174,11 +174,34 @@ const MessageBody = React.memo(({
 
   const content = isCustomHTML
     ? twemojify(sanitizeCustomHtml(body), undefined, true, false)
-    : <p>{twemojify(body, undefined, true)}</p>;
+    : twemojify(body, undefined, true);
+
+  // Determine if this message should render with large emojis
+  // Criteria:
+  // - Contains only emoji
+  // - Contains no more than six emoji
+  let emojiOnly = false;
+  if (content.type === 'img') {
+    // If this messages contains only a single (inline) image
+    emojiOnly = true;
+  } else if (content.constructor.name === 'Array') {
+    // Otherwise, it might be an array of images / texb
+
+    // Count the number of emojis
+    const nEmojis = content.filter((e) => e.type === 'img').length;
+
+    // Make sure there's no text besides whitespace
+    if (nEmojis <= 6 && content.every((element) => (
+      (typeof element === 'object' && element.type === 'img')
+      || (typeof element === 'string' && /^\s*$/g.test(element))
+    ))) {
+      emojiOnly = true;
+    }
+  }
 
   return (
     <div className="message__body">
-      <div className="text text-b1">
+      <div className={`text text-b1 ${emojiOnly ? 'emoji-only' : ''}`}>
```

Comment:
There are two issues IMO: 
1. I think 64px is too big in size.
2. Also if we use 64px for img then text-b1 line-height should also be 64px otherwise it will slightly displace emoji upward.

IMO we should just replace `text-b1` with `text-h1` which will scale everything automatically inside. but with that emoji height will be around 34px as the `variant h1` text size is 36px.

Also 34px is good height and we can also increase the number of jumbo emoji more than 6.

Reply:
Sounds good to me!  I'll get that up in just a minute

Reply:
![image](https://user-images.githubusercontent.com/38897201/147434746-2e74e43a-c5c1-470a-a4ba-8d24b8ab1ce1.png)

here's how that looks.  I set the emoji limit to be 10 emoji (which also happens to be what fluffychat uses, as i found out).